ra-tls/Makefile

```diff
@@ -78,13 +78,15 @@ wolfssl: WOLFSSL_CONFIGURE_FLAGS := --prefix=$(shell readlink -f $(PREFIX)) --en
 # then RATLS_ECDSA will be wrote into options.h, even just make(compiling attestation based on EPID), the RATLS_ECDSA will still be in
 # options.h. So it needs be reconfigured before compiling, options.h will be re-generated according to new configuration.
 wolfssl:
-	if [ -d "./wolfssl" ]; then \
-		cd wolfssl && ./autogen.sh && CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true; \
-	fi
 	if [ ! -d "./wolfssl" ]; then \
-		git clone -b v4.6.0-stable https://github.com/wolfSSL/wolfssl && \
-		cd wolfssl && git apply ../patch/wolfssl.patch && \
-		./autogen.sh && CFLAGS="$(CFLAGS)" ./configure $(WOLFSSL_CONFIGURE_FLAGS) || true; \
+		git clone -b v4.6.0-stable https://github.com/wolfSSL/wolfssl || true; \
+	fi
+	if [ -d "./wolfssl" ]; then \
```

Comment:
Why allowing the clone operation to go failure?
And if why checking whether ./wolfssl exists here?

Reply:
git clone to allow failure is a mistake.

Mainly used when clone is needed for the first time.

Reply:
> Why allowing the clone operation to go failure?

git clone to allow failure is a mistake.

> And if why checking whether ./wolfssl exists here? If not exist, the repo will be cloned, and then ./wolfssl will exist for sure, unless git clone failed.

Mainly used when clone is needed for the first time.

Reply:
I mean this condition check `if [ -d "./wolfssl" ]; then` is not necessary?

Reply:
Yes, this condition check can be deleted, I will remove it.